stripe-core/src/main/java/com/stripe/android/core/networking/RequestExecutor.kt

```diff
@@ -0,0 +1,42 @@
+package com.stripe.android.core.networking
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.core.exception.APIConnectionException
+import com.stripe.android.core.exception.APIException
+import com.stripe.android.core.model.StripeModel
+import com.stripe.android.core.model.parsers.ModelJsonParser
+import com.stripe.android.core.model.parsers.StripeErrorJsonParser
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)
+suspend fun <Response : StripeModel> executeRequestWithModelJsonParser(
```

Comment:
Took this from identity, now should be available for all the SDKs. @ccen-stripe you might want to reference this and remove it from Identity.